.travis.yml

```diff
@@ -13,6 +13,10 @@ matrix:
     - python: 3.5
       env:
         - TOX_ENV=py35
+    - env: TOX_ENV=functional
```

Comment:
This is a bit strange, shouldn't this `env` be listed with the others above, below line 6? It doesn't need this "include" special matrix hack I believe, as it'll be run with py27 (at least that's what it's doing [at the moment](https://travis-ci.org/mozilla-services/xml2kinto/jobs/116093357))